front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -249,50 +249,19 @@ const AdminProjectEventEdit = () => {
     setAttendanceLimitVisible(toggleValue);
     setAttributeDiff({
       ...attributeDiff,
-      maximum_attendees: null,
+      maximum_attendees: toggleValue === false ? null : 100,
     });
   };
 
-  const handleMaximumAttendeesChange = (value: string) => {
```

Comment:
I wouldn't do this validation on each keystroke. Firstly, because other fields in this form do not have such validation. We should ensure fields in the same form behave consistently. Secondly, because this form is already bloated and should be refactored to use our hook form solution. Adding more bloat will make it harder to refactor later (people would have to figure out what this function is doing, and why this field gets different treatment than other fields).

Instead, I opted to set the back-end errors returned after submission, as we do elsewhere in this form.